diagrams/sequence-diagrams/publishing-a-form.md

```diff
@@ -15,22 +15,20 @@ sequenceDiagram
   admin-->>user: show "Create a form" page
   user->>browser: Click "Make your form live" link
   browser->>admin: GET /forms/{form id}/make-live
-  admin->>api: GET /forms/{form id}
-  admin->>api: GET /forms/{form id}
-  admin->>api: GET /forms/{form id}
+  admin->>api: GET /api/v1/forms/{form id}
+  admin->>api: GET /api/v1/forms/{form id}
   browser-->>user: show "Make your form live" page
 
   user->>browser: Confirm making form live<br />click "Save and continue" button
   browser->>admin: POST /forms/{form id}/make-live
-  admin->>api: GET /forms/{form id}
-  admin->>api: GET /forms/{form id}
-  admin->>api: GET /forms/{form id}/pages
-  admin->>api: PUT /forms/{form id}/make-live
-  admin-->>browser: 302
+  admin->>api: GET /api/v1/forms/{form id}
+  admin->>api: GET /api/v1/forms/{form id}
```

Comment:
Was this double request resolved by https://github.com/alphagov/forms-admin/pull/332 ?
```suggestion
  admin->>api: GET /api/v1/forms/{form id}
```

Reply:
No, this is the part where I noticed there were 3 requests being made (2 of those were triggered by using `current_form`. After that PR you have mentioned it was reduced to 2 calls. 

Basically all our requests make two calls to that endpoint, the first one is for authorization check to ensure the form belongs to the org, and the next request is for the form itself. I have a ticket to have a look at these double requests as part of the firebreak.

Reply:
Ah OK - that makes sense, thanks!